platform/core.windows/src/org/netbeans/core/windows/services/DialogDisplayerImpl.java

```diff
@@ -252,11 +252,8 @@ public void showDialog () {
                         presenter = new NbPresenter(descriptor, NbPresenter.currentModalDialog, true);
                     }
                 } else {
-                    Frame f = KeyboardFocusManager.getCurrentKeyboardFocusManager().getActiveWindow() 
-                        instanceof Frame ? 
-                        (Frame) KeyboardFocusManager.getCurrentKeyboardFocusManager().getActiveWindow() 
-                        : WindowManager.getDefault().getMainWindow();
-
+                    Window w = KeyboardFocusManager.getCurrentKeyboardFocusManager ().getActiveWindow ();
+                    Frame f = w instanceof Frame ? (Frame) w : WindowManager.getDefault().getMainWindow();
```

Comment:
This is a cosmetic change. It's both more clear and allows easier comparison to what's going on in the
```
if (descriptor instanceof DialogDescriptor) {
```
case